### ISSUE TITLE
Conduct screen reader testing with widget in staging and troubleshoot

### ISSUE THREAD

**Issue by tameracorson** [department-of-veterans-affairs/va.gov-team#43485]
- [x] Conduct screen reader testing of flatten data from widget
- [x] Conduct screen reader testing of unflatten data from widget
- [x] Conduct screen reader testing of dat from widget without PII fields
- [x] Determine if PDF widget is repairable to present accessible solution

**Comment by tameracorson** [comment]
![image.png](https://images.zenhubusercontent.com/61856bab21cd5965714d703d/ce185ba8-20e6-4439-a9fa-3b32e8795bfe)

**Comment by tameracorson** [comment]
https://images.zenhubusercontent.com/133843125/99779d49-1c5a-41d7-b1f0-a7ec7a44cbfe/video_of_unflat_version_6_30_on_screen_read_test_7_7_.mp4[https://app.zenhub.com/files/133843125/0120aa12-0f61-42bc-b477-9454b596a4c5/download](https://app.zenhub.com/files/133843125/0120aa12-0f61-42bc-b477-9454b596a4c5/download)

**Comment by tameracorson** [comment]
![image.png](https://images.zenhubusercontent.com/61856bab21cd5965714d703d/3318ca64-0ead-4960-997c-ed4b2c4cb336)

**Comment by tameracorson** [comment]
![image.png](https://images.zenhubusercontent.com/61856bab21cd5965714d703d/1e1d7361-540c-47c2-b127-2c5bc169c2f9)

**Comment by tameracorson** [comment]
**Testing round with Hexapdf**

**Understanding Jaws reader:**
-A red box follows on PDF where it is reading
-Each table cell should be read like this in Jaws
o	Read only 4 Street address left paren including apartment no. right paren
o	4 Street address left paren including apartment no. right paren
o	Edit 123 test st

**Test #1 (hexa_untouched_1095b)**
-Passes adobe accessibility screen
-Completed all 3 cell requirements above where content was present
-Completed first 2 cell requirements on blank cells except…
-In name cells, only part 1 above read for cells 1 and 23a, 23b, 23c

**Test #2 (hexa_readonly_1095b)**
-Passes adobe accessibility screen
-Completed all 3 cell requirements above where content was present; added word read only before saying edit
-Completed first 2 cell requirements on blank cells with the added wording of read only before saying edit except…
-In name cells, only part 1 above read for cells 1 and 23a, 23b, 23c

Test #3 (hexa_flattened-1095b)
-Passes adobe accessibility screen
-Only proceeded to complete 1 cell requirement for all cells, did not read any content
-At box 23, instead of going across it jumped down to 24, then 25, 26, etc…

**Comment by tameracorson** [comment]
[https://app.zenhub.com/files/133843125/9f8e7b45-832b-4616-89d3-0b08af2718f1/download](https://app.zenhub.com/files/133843125/9f8e7b45-832b-4616-89d3-0b08af2718f1/download)[https://app.zenhub.com/files/133843125/5d2dd8c6-f864-490f-a728-b3eee53251db/download](https://app.zenhub.com/files/133843125/5d2dd8c6-f864-490f-a728-b3eee53251db/download)[Uploading... hexa_flattened_1095b.pdf]

**Comment by tameracorson** [comment]
See DSVA slack for video
